quizzes/quiz_2.md

```diff
@@ -0,0 +1,64 @@
+
+{quiz, id: creating_plan, attempts: 10}
+
+## Creating a DMS Plan Quiz
+
+Choose the best answer for the following questions.
+
+{choose-answers: 4}
+? Which is NOT a major section of the NIH DMS plan?
+
+C) Data interpretation - how data will be used in follow-up studies and other research applications
+C) Data practices - how your data analysis fits within the current data science ecosystem
+o) Data type - describe what data (amount and type) will be generated over the course of funding and what data will or will not be shared
+o) Tools, software, and code - describe what tools (and versions) you intend to use to manage and analyze the data (note code is not required to be shared)
+o) Standards - describe any standards that you might need to use for your data and metadata to make them usable by others or be contributed to a repository
+o) Data preservation, access, timelines - describe where the data will be made available and when
+o) Access, distribution, reuse considerations - describe how you have carefully considered any reasons that might limit sharing
+o) Oversight - describe who will manage compliance of the DMS plan
+
+{choose-answers: 4}
+? What factors influence your data storage needs?
+
+C) The type of data and the number of samples
+m) The number of researchers working on the project
+m) The location of the data storage facility
+m) The funding agency's requirements for data storage
```

Comment:
m) The funding agency's requirements for data storage ...I think this answer could be correct too actually- like that your agency requires that you share all raw data for example

Reply:
Good point!